dev/src/convert.ts

```diff
@@ -67,14 +67,14 @@ function convertTimestamp(timestampValue, argumentName?) {
     }
 
     timestampProto = {
-      seconds: seconds || undefined,
-      nanos: nanos || undefined,
+      seconds,
+      nanos,
     };
   } else if (is.defined(timestampValue)) {
     validate.isObject('timestampValue', timestampValue);
     timestampProto = {
-      seconds: timestampValue.seconds || undefined,
-      nanos: timestampValue.nanos || undefined,
+      seconds: timestampValue.seconds,
+      nanos: timestampValue.nanos,
```

Comment:
I think that this change my break the conformance tests (either now or in the future), which require the absolute minimum proto. ` timestamp { seconds: 0, nanos: 0 }` might fail tests that require `timestamp {} ` (in JSON format, not Proto3).

This might be totally wrong, but it's the only explanation I could come up with why I might have written this code this way.